docs/reference/function-configuration/function-configuration-reference.md

```diff
@@ -85,6 +85,7 @@ The `spec` section contains the requirements and attributes and has the followin
 | ~~triggers.(name).maxWorkers~~                                       | int                                                                                                        | **Deprecated:** The max number of concurrent requests this trigger can process                                                                                                                                                                                                                                    |
 | triggers.(name).kind                                                 | string                                                                                                     | The trigger type (kind) - `cron` \ `eventhub` \ `http` \ `kafka-cluster` \ `kinesis` \ `nats` \ `rabbit-mq`                                                                                                                                                                                                       |
 | triggers.(name).url                                                  | string                                                                                                     | The trigger specific URL (not used by all triggers)                                                                                                                                                                                                                                                               |
+| triggers.(name).workerTerminationTimeout                             | string                                                                                                     | Waiting time for workers to drop or ACK on events before rebalance in seconds or as a duration string (e.g., `5s`, `1m`, `1h`); `10s` by default                                                                                                                                                                  |
```

Comment:
Is this timeout only related to Kafka?
If so, move it to the kafka doc.
If not - then ACK/Rebalanced shouldn't define it...

Reply:
@TomerShor Now it's used only in kafka, but what my idea is that all parameters which are global and can be effective for all triggers should be in this docs. I guess the mistake here is that this configuration wasn't put to `triggers.(name).attributes` , but since it's here already we should mention it in both places

Reply:
OK I see you point.
At the very least met ion that it is related to Kafka trigger.
Also, we can add a TODO to move this to attributes